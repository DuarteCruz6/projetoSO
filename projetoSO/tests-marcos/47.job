WRITE [(w4,20)(x5,98)]
DELETE [w4,x5]
WRITE [(g1,80)(j1,27)(t9,67)]
READ [t9]
WRITE [(o1,89)(w2,91)(r5,89)(f9,94)(y8,53)]
DELETE [f9,o1,y8,j1,t9]
WRITE [(w5,30)(x9,71)]
READ [g1]
WRITE [(t3,19)(q0,74)(p1,79)(a6,88)]
WRITE [(t5,62)(o6,65)(t1,39)(j7,3)(v8,73)]
