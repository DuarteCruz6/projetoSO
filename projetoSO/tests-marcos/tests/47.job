WRITE [(b4,70)(y3,43)(v2,20)(j3,53)(l1,10)]
WRITE [(e8,57)(c4,45)(y2,35)(c9,39)(x9,20)]
READ [y3]
DELETE [x9]
WRITE [(n7,8)(i9,12)]
READ [y3]
DELETE [b4,y2]
DELETE [i9,c9,e8]
WRITE [(v0,97)]
READ [v2]
WRITE [(d9,8)(x2,13)(m3,80)(o7,77)(t5,75)]
READ [d9]
WRITE [(x5,43)(j4,81)(c3,81)(h7,72)]
DELETE [y3,l1,m3]
WRITE [(g0,87)(x1,6)]
WRITE [(r6,33)(a6,25)]
WRITE [(r1,25)(k9,34)(h1,0)(s7,91)]
WRITE [(e5,18)(z9,69)(t0,18)]
WRITE [(j1,82)(q3,34)(n9,35)(o4,29)(a2,74)]
WRITE [(s2,72)(w4,64)(g7,10)(y3,35)]
BACKUP
SHOW
DELETE [n9]
WRITE [(y6,46)(x7,75)]
READ [v2]
WRITE [(u7,27)]
WRITE [(q4,43)(r4,85)]
SHOW
WRITE [(k5,8)(n6,46)(p0,13)(d5,11)(j9,97)]
WRITE [(h6,49)]
READ [h7]
READ [r6]
READ [v2]
WRITE [(d0,54)(k1,16)(p5,99)(u9,66)(a4,71)]
DELETE [c4]
SHOW
READ [s7]
WRITE [(r3,100)]
WRITE [(i1,4)(e4,47)(y4,85)]
WRITE [(b9,32)]
READ [y6]
BACKUP
WRITE [(p9,31)(p7,14)(i6,32)(k7,94)]
READ [y4]
BACKUP
READ [e4]
READ [p0]
WRITE [(f0,53)(a9,52)(v6,78)(n5,63)(y2,92)]
WRITE [(d4,82)]
WRITE [(o6,82)(n0,27)(l2,99)(m3,0)(g8,55)]
SHOW
WRITE [(i2,65)]
READ [a6]
WRITE [(g5,92)(n9,37)(s4,84)]
READ [m3]
WAIT 1000
READ [n9]
WRITE [(k6,72)]
READ [p5]
SHOW
WRITE [(s6,73)(u0,36)(q7,69)]
WRITE [(r7,60)(f4,28)]
READ [c3]
BACKUP
DELETE [y6,k6,n0,e5]
DELETE [j3,f4,x7,r1,u9]
WRITE [(m4,77)(q6,43)(z6,7)]
WAIT 1000
WAIT 1000
