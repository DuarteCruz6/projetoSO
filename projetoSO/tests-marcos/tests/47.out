[(y3,43)]
[(y3,43)]
[(v2,20)]
[(d9,8)]
(a2, 74)
(a6, 25)
(c3, 81)
(c4, 45)
(d9, 8)
(e5, 18)
(g7, 10)
(g0, 87)
(h1, 0)
(h7, 72)
(j1, 82)
(j4, 81)
(j3, 53)
(k9, 34)
(n9, 35)
(n7, 8)
(o4, 29)
(o7, 77)
(q3, 34)
(r1, 25)
(r6, 33)
(s2, 72)
(s7, 91)
(t0, 18)
(t5, 75)
(v0, 97)
(v2, 20)
(w4, 64)
(x1, 6)
(x5, 43)
(x2, 13)
(y3, 35)
(z9, 69)
[(v2,20)]
(a2, 74)
(a6, 25)
(c3, 81)
(c4, 45)
(d9, 8)
(e5, 18)
(g7, 10)
(g0, 87)
(h1, 0)
(h7, 72)
(j1, 82)
(j4, 81)
(j3, 53)
(k9, 34)
(n7, 8)
(o4, 29)
(o7, 77)
(q4, 43)
(q3, 34)
(r4, 85)
(r1, 25)
(r6, 33)
(s2, 72)
(s7, 91)
(t0, 18)
(t5, 75)
(u7, 27)
(v0, 97)
(v2, 20)
(w4, 64)
(x7, 75)
(x1, 6)
(x5, 43)
(x2, 13)
(y6, 46)
(y3, 35)
(z9, 69)
[(h7,72)]
[(r6,33)]
[(v2,20)]
(a4, 71)
(a2, 74)
(a6, 25)
(c3, 81)
(d0, 54)
(d5, 11)
(d9, 8)
(e5, 18)
(g7, 10)
(g0, 87)
(h6, 49)
(h1, 0)
(h7, 72)
(j9, 97)
(j1, 82)
(j4, 81)
(j3, 53)
(k1, 16)
(k5, 8)
(k9, 34)
(n6, 46)
(n7, 8)
(o4, 29)
(o7, 77)
(p5, 99)
(p0, 13)
(q4, 43)
(q3, 34)
(r4, 85)
(r1, 25)
(r6, 33)
(s2, 72)
(s7, 91)
(t0, 18)
(t5, 75)
(u9, 66)
(u7, 27)
(v0, 97)
(v2, 20)
(w4, 64)
(x7, 75)
(x1, 6)
(x5, 43)
(x2, 13)
(y6, 46)
(y3, 35)
(z9, 69)
[(s7,91)]
[(y6,46)]
[(y4,85)]
[(e4,47)]
[(p0,13)]
(a9, 52)
(a4, 71)
(a2, 74)
(a6, 25)
(b9, 32)
(c3, 81)
(d4, 82)
(d0, 54)
(d5, 11)
(d9, 8)
(e4, 47)
(e5, 18)
(f0, 53)
(g8, 55)
(g7, 10)
(g0, 87)
(h6, 49)
(h1, 0)
(h7, 72)
(i6, 32)
(i1, 4)
(j9, 97)
(j1, 82)
(j4, 81)
(j3, 53)
(k7, 94)
(k1, 16)
(k5, 8)
(k9, 34)
(l2, 99)
(m3, 0)
(n0, 27)
(n5, 63)
(n6, 46)
(n7, 8)
(o6, 82)
(o4, 29)
(o7, 77)
(p7, 14)
(p9, 31)
(p5, 99)
(p0, 13)
(q4, 43)
(q3, 34)
(r3, 100)
(r4, 85)
(r1, 25)
(r6, 33)
(s2, 72)
(s7, 91)
(t0, 18)
(t5, 75)
(u9, 66)
(u7, 27)
(v6, 78)
(v0, 97)
(v2, 20)
(w4, 64)
(x7, 75)
(x1, 6)
(x5, 43)
(x2, 13)
(y2, 92)
(y4, 85)
(y6, 46)
(y3, 35)
(z9, 69)
[(a6,25)]
[(m3,0)]
[(n9,37)]
[(p5,99)]
(a9, 52)
(a4, 71)
(a2, 74)
(a6, 25)
(b9, 32)
(c3, 81)
(d4, 82)
(d0, 54)
(d5, 11)
(d9, 8)
(e4, 47)
(e5, 18)
(f0, 53)
(g5, 92)
(g8, 55)
(g7, 10)
(g0, 87)
(h6, 49)
(h1, 0)
(h7, 72)
(i2, 65)
(i6, 32)
(i1, 4)
(j9, 97)
(j1, 82)
(j4, 81)
(j3, 53)
(k6, 72)
(k7, 94)
(k1, 16)
(k5, 8)
(k9, 34)
(l2, 99)
(m3, 0)
(n9, 37)
(n0, 27)
(n5, 63)
(n6, 46)
(n7, 8)
(o6, 82)
(o4, 29)
(o7, 77)
(p7, 14)
(p9, 31)
(p5, 99)
(p0, 13)
(q4, 43)
(q3, 34)
(r3, 100)
(r4, 85)
(r1, 25)
(r6, 33)
(s4, 84)
(s2, 72)
(s7, 91)
(t0, 18)
(t5, 75)
(u9, 66)
(u7, 27)
(v6, 78)
(v0, 97)
(v2, 20)
(w4, 64)
(x7, 75)
(x1, 6)
(x5, 43)
(x2, 13)
(y2, 92)
(y4, 85)
(y6, 46)
(y3, 35)
(z9, 69)
[(c3,81)]
